soh/soh/Enhancements/randomizer/3drando/entrance.cpp

```diff
@@ -81,12 +81,25 @@ void SetAllEntrancesData(std::vector<EntranceInfoPair>& entranceShuffleTable) {
     forwardEntrance->SetBlueWarp(forwardEntry.blueWarp);
     forwardEntrance->SetType(forwardEntry.type);
     forwardEntrance->SetAsPrimary();
+
+    // When decouple entrances is on, mark it for everything except boss rooms so 
```

Comment:
> everything except boss rooms so

looks like you didn't finish writing the comment here, so what?

Reply:
I'll fix that, and a typo in another comment